### PR TITLE
implement devise jwt registration/authorization

### PR DESCRIPTION
**Pull request by Bilanuk**
https://medium.com/@alessandrahagarty/using-devise-for-authentication-in-a-react-rails-app-f6a0eb87fbd5
https://devsday.ru/blog/details/52846
image

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :null_session, if: proc {|c| c.request.format == "application/json" }
```
Comment (Bilanuk) [inline app/controllers/application_controller.rb:4]: will configure cors in next pull request

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (ghost) [inline test/fixtures/users.yml]: why do we need this file

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```
Comment (ghost) [inline test/models/user_test.rb]: same here, use it or remove

**Review by ghost — CHANGES_REQUESTED** [review]

File: test/fixtures/users.yml (hunk 2 above)
Comment (Bilanuk) [reply test/fixtures/users.yml]: no bitches

**Review by ghost — APPROVED** [review]

**Pull request merged**